internal/pkg/manifest/errors.go

```diff
@@ -7,12 +7,12 @@ import (
 	"fmt"
 )
 
-// ErrInvalidSvcManifestType occurs when a user requested a manifest template type that doesn't exist.
-type ErrInvalidSvcManifestType struct {
+// ErrInvalidWkldManifestType occurs when a user requested a manifest template type that doesn't exist.
+type ErrInvalidWkldManifestType struct {
```

Comment:
nit: we can remove "manifest" from the name so that it reads `manifest.ErrInvalidWorkloadType`.
```suggestion
// ErrInvalidWorkloadType occurs when a user requested a manifest template type that doesn't exist.
type ErrInvalidWorkloadType struct {
```